### PR TITLE
update pricing page

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description of the change

image

image

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/data/pricing.ts [hunk 1]
```diff
@@ -26,67 +26,68 @@ export const pricingMatrix = [
...
+    business: "Google + Github",
   },
   {
-    label: "Data Export",
-    standard: false,
-    enterprise: true,
+    label: "GPDR Compliant",
+    developer: "",
+    pro: "",
+    business: "Yes",
```
Comment (troygoode) [inline src/data/pricing.ts:91]: lets change the label to "Data Processing Agreement (DPA)"

File: src/data/pricing.ts [hunk 2]
```diff
@@ -26,67 +26,68 @@ export const pricingMatrix = [
...
   },
   {
-    label: "HIPAA Compliance",
-    standard: false,
-    enterprise: true,
+    label: "Log Retention",
+    developer: "7 Days",
+    pro: "30 Days",
+    business: "30 Days",
```
Comment (troygoode) [inline src/data/pricing.ts:79]: Developer: 48 Hours
Pro: 7 Days
Business: 30 Days

File: src/data/pricing.ts (hunk 1 above)
Comment (troygoode) [reply src/data/pricing.ts:91]: actually, inspired by OneSignal:

```
label: Data Privacy
dev: GDPR & CCPA
pro: GDPR & CCPA w/ DPA
bus: GDPR & CCPA w/ DPA
```

File: src/data/pricing.ts (hunk 2 above)
Comment (rileylnapier) [reply src/data/pricing.ts:79]: damn, 48 hours

**Pull request merged**